plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/CodeTemplatesTest.java

```diff
@@ -108,11 +108,11 @@ public void testMaterializeAppEngineStandardFiles_objectifyWithJava8()
       throws CoreException, ParserConfigurationException, SAXException, IOException {
     AppEngineProjectConfig config = new AppEngineProjectConfig();
     config.setRuntime(AppEngineRuntime.STANDARD_JAVA_8);
-    config.setAppEngineLibraries(Collections.singleton(new Library("objectify"))); // Objectify 5
+    config.setAppEngineLibraries(Collections.singleton(new Library("objectify")));
 
     CodeTemplates.materializeAppEngineStandardFiles(project, config, monitor);
     assertTrue(objectifyFilterClassExists());
-    assertFalse(objectifyListenerClassExists()); // listener added only for Objectify 6
+    assertTrue(objectifyListenerClassExists());
```

Comment:
If we add other instances of library-version-specific code-generation, we should make the tests be (somehow) attuned to the library version.